src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -1111,6 +1115,7 @@ function IOURequestStepConfirmation({
                         iouMerchant={transaction?.merchant}
                         iouCreated={transaction?.created}
                         isDistanceRequest={isDistanceRequest}
+                        isManualDistanceRequest={isManualDistanceRequest}
```

Comment:
It would be better if we could rename these as: `isMapDistanceRequest` and `isManualDistanceRequest`.